Sources/KeyboardShortcuts/Shortcut.swift

```diff
@@ -292,3 +294,59 @@ extension KeyboardShortcuts.Shortcut: CustomStringConvertible {
 		modifiers.description + (keyToCharacter()?.uppercased() ?? "�")
 	}
 }
+
+// MARK: - SwiftUI Helpers
+
+extension KeyboardShortcuts.Shortcut {
+  
+  @available(iOS 14.0, macOS 11.0, *)
+  var swiftUI: SwiftUI.KeyboardShortcut {
+    .init(.init(keyEquivalent.first!), modifiers: modifiers.swiftUI)
+  }
+
+}
+
+@available(macOS 10.15, *)
+extension NSEvent.ModifierFlags {
+  var swiftUI: SwiftUI.EventModifiers {
+    var modifiers: SwiftUI.EventModifiers = []
+    if contains(.shift) {
+      modifiers.insert(.shift)
+    }
+    if contains(.command) {
+      modifiers.insert(.command)
+    }
+    if contains(.capsLock) {
+      modifiers.insert(.capsLock)
+    }
+    if contains(.function) {
+      modifiers.insert(.function)
+    }
+    if contains(.option) {
+      modifiers.insert(.option)
+    }
+    if contains(.control) {
+      modifiers.insert(.control)
+    }
+    return modifiers
+  }
+}
+
+@available(iOS 14.0, macOS 11.0, *)
+extension View {
+  
+  @ViewBuilder
+  /// Assigns the global keyboard shortcut to the modified control.
+  ///
+  /// Only assigns a keyboard shortcut, if one was defined (or it has a default shortcut).
+  ///
+  /// - Parameter shortcut: Strongly-typed name of the shortcut
+  public func keyboardShortcut(_ shortcut: KeyboardShortcuts.Name) -> some View {
```

Comment:
If the shortcut attached to `KeyboardShortcuts.Name` is changed by the user, it will not be changed here. You need to use a `ViewModifier` and listen to updates to the keyboard shortcut, like the `NSMenuItem` extension does.

Reply:
I expected so, too, but that's not necessary in my testing. As soon as I assign a new keyboard shortcut using the `Recorder`, the `.commands` is re-evaluated as this method will be called again.

Reply:
I think that only applies if they're in the same view. But that's not the case if they are in different windows (preferences vs main app window).